modules/sdk/project-templates-npm-billboardjs-portlet/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -0,0 +1,40 @@
+#parse( "definitions.vm" )
+
+<%@ include file="/init.jsp" %>
+
+<!-- Temporary workaround to obtain the library stylesheets -->
+<link href="/o/${artifactId}/node_modules/billboard.js@1.1.1/dist/billboard.css" rel="stylesheet">
+
+<aui:script require="${artifactId}@${packageJsonVersion}">
+	${auiScriptRequireVarName}.default('<portlet:namespace />');
+</aui:script>
+
+<hr>
+
+<div id="<portlet:namespace />-billboard">
```

Comment:
@izaera: AFAIK, IDs don't have a dash after the portlet namespace. Is it really necessary here?

Reply:
It's not totally necessary, but it's not illegal in HTML. I would say it is even common. But if you want, we can remove them.

Reply:
It's just to be consistent with Liferay's convention. At least, I think it's a convention 😅